### PR TITLE
[FIX] manifestCreator: set fallbackLocale to empty string if no locale is present

### PR DESCRIPTION
**Pull request by flovogt**
Instead of using a artificial placeholder, you use a real message bundle file for tests.

### CODE REVIEW

File: test/expected/build/library.i/bundle-library.h-build-manifest/resources/library/i/manifest.json [hunk 1]
```diff
@@ -4,7 +4,12 @@
     "id": "library.i",
     "type": "library",
     "embeds": [],
-    "i18n": "(i18nPathRelativeToLibrary)",
+    "i18n": {
+      "bundleUrl": "i18nf/custommessagebundle.properties",
+      "supportedLocales": [
+        ""
+      ]
```
Comment (flovogt) [inline test/expected/build/library.i/bundle-library.h-build-manifest/resources/library/i/manifest.json:11]: @codeworrior in this case we should set the fallback locale to `""` isn't?

File: lib/processors/manifestCreator.js [hunk 2]
```diff
@@ -560,9 +560,16 @@ async function createManifest(
 
 		const supportedLocalesArray = Array.from(supportedLocales);
 		supportedLocalesArray.sort();
+
+		let fallbackLocale;
+		if (supportedLocales.size === 1 && supportedLocales.has("")) {
+			fallbackLocale = "";
```
Comment (flovogt) [inline lib/processors/manifestCreator.js:566]: size 1 --> set fallbackLocale to the one entry

Comment (flovogt) [reply lib/processors/manifestCreator.js]: done

File: test/lib/tasks/generateLibraryManifest.js [hunk 3]
```diff
@@ -214,7 +215,154 @@ test("integration: Library with i18n=true declared in .library", async (t) => {
...
-					supportedLocales: [""]
+					supportedLocales: [""],
+					fallbackLocale: ""
+				}
+			}
+		},
+	});
+});
+
+test("integration: Library with i18n=true declared in .libraryb and multiple locales", async (t) => {
```
Comment (matz3) [inline test/lib/tasks/generateLibraryManifest.js:226]: .libraryb?

File: test/lib/tasks/generateLibraryManifest.js [hunk 4]
```diff
@@ -214,7 +215,154 @@ test("integration: Library with i18n=true declared in .library", async (t) => {
...
+				i18n: {
+					bundleUrl: "messagebundle.properties",
+					supportedLocales: ["", "en"]
+				}
+			}
+		},
+	});
+});
+
+test("integration: Library with i18n=true declared in .libraryb and single locale", async (t) => {
```
Comment (matz3) [inline test/lib/tasks/generateLibraryManifest.js:301]: .libraryb?

Comment (flovogt) [reply test/lib/tasks/generateLibraryManifest.js]: thx... typo

File: test/lib/tasks/generateLibraryManifest.js (hunk 3 above)
Comment (flovogt) [reply test/lib/tasks/generateLibraryManifest.js]: done

**Review by matz3 — APPROVED** [review]

**Pull request merged**